### ISSUE TITLE
Did you get the same result as the author???

### ISSUE THREAD

**Issue by zhuixunforever** [shen1994/DeepFaceRestruction#1]
Hi, thank you for your contribution!!
I am using your code for training, but I did not get the ideal result. Did you get good result before?

**Comment by shen1994** [comment]
yes, result is good


------------------
沈新锋


                     此致
                                  敬礼！


SHEN Xin-feng, M.S., Postgraduate
Acdemy of Computer & Software
Nanjing University of Information Science & Technology
No.219, Ningliu Road, Nanjing, Jiangsu Province, China (Postcode: 210044)
Phone: 13182838068

Email: 843224306@qq.com/sxf_0319@163.com
 

 




------------------ 原始邮件 ------------------
发件人: "zhuixunforever"<notifications@github.com>;
发送时间: 2019年6月6日(星期四) 中午11:35
收件人: "shen1994/DeepFaceRestruction"<DeepFaceRestruction@noreply.github.com>;
抄送: "Subscribed"<subscribed@noreply.github.com>;
主题: [shen1994/DeepFaceRestruction] Did you get the same result as theauthor??? (#1)




Hi, thank you for your contribution!!
 I am using your code for training, but I did not get the ideal result. Did you get good result before?

**Comment by tiancaipipi110** [comment]
@shen1994  Just loaded the trained model in PRNet, and got two errors
```
Not found: Key resfcn256/Conv/BatchNorm/beta not found in checkpoint
NotFoundError: Key _CHECKPOINTABLE_OBJECT_GRAPH not found in checkpoint
```
Am I missing anything?

**Comment by zhuixunforever** [comment]
@tiancaipipi110  I think you should retrain this code, rather than load the pretrained model of origin PRNet.

**Comment by tiancaipipi110** [comment]
@zhuixunforever Yes, I retrained the model using this code, and load the model to original PRNet to run `run_basics.py` to test, and it's throwing me error. Think it was because of variable name, the errors are gone after I changed `variable_scope` to `name_scope` in<a href = 'https://github.com/YadiraF/PRNet/blob/master/predictor.py#L31'> their predictor</a> to match training code. 

However now I get `ValueError: Variable shortcut/weights already exists, disallowed. Did you mean to set reuse=True or reuse=tf.AUTO_REUSE in VarScope?` , and if I set it to `reuse=tf.AUTO_REUSE` in <a href = 'https://github.com/shen1994/DeepFaceRestruction/blob/master/model.py#L17'>here</a>, it'll throw another error `ValueError: Trying to share variable shortcut/weights, but specified shape (1, 1, 32, 64) and found shape (1, 1, 16, 32).`

**Comment by zhuixunforever** [comment]
@tiancaipipi110 You're in big trouble. Sorry, I can't give you the answer because I'm a tensorflow beginner. I advice that you can write your own test.py that suited to this code, I know it's easier

**Comment by tiancaipipi110** [comment]
@zhuixunforever Thanks. Let me see.

**Comment by JianBin-Liu** [comment]
what's the mm3d's function in prepare.py file?
